src/utils/LockFile.ts

```diff
@@ -1,9 +1,9 @@
 import { LockFile } from '../types'
 
+// TODO add a link to documentation
 const header = `# IF THERE'S A GIT CONFLICT IN THIS FILE, DON'T SOLVE IT MANUALLY!
 # INSTEAD EXECUTE \`prisma migrate fix\`
```

Comment:
Uh, this should not exist any more I think. @timsuchanek ?

Reply:
You mean the "fix" command? Well then maybe we could just change it to:

```
# Prisma Migrate lockfile v1
```

Reply:
Yes, Tim and me had a conversation about this recently and also from PRs I was under the impression that this was already removed 🤷

Reply:
ping @timsuchanek @Jolg42

Reply:
I will do a cleanup, it looks like it's still there!